src/pages/reader/pdf.vue

```diff
@@ -5,17 +5,13 @@
         <div class="flex items-center gap-4">
           <NuxtLink
             :to="
-              localeLocation(
-                getAddress
-                  ? {
-                      name: 'id',
-                      params: { id: getAddress },
-                      query: { tab: 'collected' },
-                    }
-                  : { name: 'index' }
```

Comment:
How about change the check to `getSessionWallet`, if exists go to `bookshelf`, if not go to `classId` page?

Reply:
Updated